dnf/base.py

```diff
@@ -1289,16 +1293,20 @@ def install(self, pkg_spec, reponame=None):
                 self._goal.install(a)
             return len(available)
         elif self.conf.multilib_policy == "best":
-            sltr = subj.get_best_selector(self.sack)
-            if not sltr.matches():
+            sltrs = subj.get_best_selectors(self.sack)
+            match = reduce(lambda x,y: y.matches() or x, sltrs, [])
```

Comment:
add space e.g.: `x, y`. `all` function in this case would be more readable to me but thats up to you.